nova-app/src/app/styles/theme/dark.css

```
@@ -46,6 +46,7 @@
     --stroke-data-selected: var(--color-dark-full);
 
     /* ---------- Shadow ---------- */
+    --shadow-modal: 0 0 12px rgba(255, 255, 255, 0.2), 0 6px 16px rgba(0, 0, 0, 0.35);
```

Stylelint (17.2.0)

[error] Expected "0.2" to be "20%" (alpha-value-notation)

(alpha-value-notation)